www/prod/static/502.html

```diff
@@ -0,0 +1,40 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Katana</title>
+        <link rel="shortcut icon" href="favicon.ico">
+        <link rel="stylesheet" href="/prod/css/default.css" type="text/css">
```

Comment:
A user gets 502 error when the backend is down, but proxy (e.g. nginx or apache) is up. But as I know, /prod is served by our backend, so in a real case, it also raises 502 error. So in concluding, we must separate those files from the backend.